### PR TITLE
Update ports info

### PR DESCRIPTION
**Pull request by bamurtaugh**
Update based on latest views and commands.

Fixes https://github.com/microsoft/vscode-remote-release/issues/7227

### CODE REVIEW

**Review by alexr00 — APPROVED** [review]
Thanks for the update!

**Comment by bamurtaugh** [comment]
Thanks @alexr00! 

@gregvanl this ready for your review and merge whenever you're ready 😄

File: docs/remote/ssh.md [hunk 1]
```diff
@@ -236,7 +236,7 @@ If you would like the local port of the tunnel to be different than the remote s
 
 Right-click the tunnel you want to modify, and select **Change Local Port** in the context menu.
```
Comment (gregvanl) [inline docs/remote/ssh.md:237]: Should this be "Change Local Address Port" to match the content menu item?

File: docs/remote/ssh.md [hunk 2]
```diff
@@ -214,13 +214,13 @@ Sometimes when developing, you may need to access a port on a remote machine tha
 
 ### Temporarily forwarding a port
 
-Once you are connected to a host, if you want to **temporarily forward** a new port for the duration of the session, select **Forward a Port** from the Command Palette (`kbstyle(F1)`, `kb(workbench.action.showCommands)`) or click on the **Forward New Port** icon in the **Remote Explorer** after selecting it from the Activity Bar.
+Once you are connected to a host, if you want to **temporarily forward** a new port for the duration of the session, select **Forward a Port** from the Command Palette (`kbstyle(F1)`, `kb(workbench.action.showCommands)`) or click on the **Add Port** button in the **Ports view**. You can see the Ports view in the bottom panel, or by running the command **Ports: Focus on Ports View**.
```
Comment (gregvanl) [inline docs/remote/ssh.md:217]: Suggest: "select the **Add Port** button..."

File: docs/remote/ssh.md [hunk 3]
```diff
@@ -214,13 +214,13 @@ Sometimes when developing, you may need to access a port on a remote machine tha
...
 
-![Remote Explorer forward port button](images/ssh/ssh-explorer-forward-port.png)
```
Comment (gregvanl) [inline docs/remote/ssh.md:219]: Can you also delete the three unused images in this same PR

File: docs/remote/ssh.md (hunk 2 above)
Comment (bamurtaugh) [reply docs/remote/ssh.md:217]: Done!

File: docs/remote/ssh.md (hunk 3 above)
Comment (bamurtaugh) [reply docs/remote/ssh.md:219]: Just deleted!

File: docs/remote/ssh.md (hunk 1 above)
Comment (bamurtaugh) [reply docs/remote/ssh.md]: I think so, just changed.

**Comment by bamurtaugh** [comment]
Just addressed all feedback, thanks for the comments @gregvanl!

**Comment by gregvanl** [comment]
Thanks!

**Pull request merged**